File: src/main/java/aquarius0715/commandcountplugin/Events.java
package aquarius0715.commandcountplugin;

import org.bukkit.event.EventHandler;
import org.bukkit.event.Listener;
import org.bukkit.event.player.PlayerJoinEvent;

public class Events implements Listener {

    CommandCountPlugin plugin;

    public Events(CommandCountPlugin plugin) {
        this.plugin = plugin;
    }

    @EventHandler
    public void onJoin(PlayerJoinEvent event) {
        if (plugin.pluginStats && !plugin.joinOnTheWay) {
            event.getPlayer().sendMessage("只今レイドが行われています。");
            event.getPlayer().setScoreboard(plugin.scoreboard);
        }
    }


}

File: src/main/java/aquarius0715/commandcountplugin/Events.kt
package aquarius0715.commandcountplugin

import org.bukkit.event.EventHandler
import org.bukkit.event.Listener
import org.bukkit.event.player.PlayerJoinEvent

class Events(var plugin: CommandCountPlugin) : Listener {
    @EventHandler
    fun onJoin(event: PlayerJoinEvent) {
        if (plugin.gameStats) {
            event.player.sendMessage(plugin.prefix + "只今レイドが行われています。")
            plugin.scoreBoardData.updateScoreBoard()
            event.player.scoreboard = plugin.scoreboard!!
        }
    }
}